src/scr.c

```diff
@@ -754,6 +812,11 @@ static int scr_get_params()
     scr_username = scr_env_username();
   }
 
+  if (scr_my_rank_world == 0) {
```

Comment:
Where possible let's bump these print blocks up to be adjacent to the block where the value is set (no empty line in between).
```
if ((value = scr_param_get(...)) != NULL) {
  scr_username = ...
  ...
}
if (scr_my_rank_world == 0) {
 ...
}
```
That makes it less likely someone will put some other unrelated code in between those two blocks.